src/applications/personalization/account/containers/AccountApp.jsx

```diff
@@ -1,17 +1,29 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
 import backendServices from 'platform/user/profile/constants/backendServices';
 import { selectUser, isLOA3 } from 'platform/user/selectors';
 
 import AccountMain from '../components/AccountMain';
 import RequiredLoginView from 'platform/user/authorization/components/RequiredLoginView';
+import localStorage from 'platform/utilities/storage/localStorage';
 import DowntimeNotification, {
   externalServices,
 } from 'platform/monitoring/DowntimeNotification';
 
 import { fetchMHVAccount } from 'platform/user/profile/actions';
 
+import { selectShowProfile2 } from 'applications/personalization/profile-2/selectors';
+
 function AccountApp(props) {
+  useEffect(
+    () => {
+      if (props.redirect) {
```

Comment:
Could we handle the redirect in `ProfilesWrapper`? 😃

Reply:
this component lives at a different route, `/account`, so not really :-)

Reply:
Got it! Ha, I was confused. I tried making it work locally but then I realized the user would never hit the `ProfilesWrapper`.. I get it now. It would still be really nice if we could use the same `Redirect` from `react-router-dom`, but that would require a bigger re-work.

Reply:
yeah and this is temporary. when we launch Profile 2.0, this component will be killed and we'll do a proper 301 redirect on the backend.